playwright.config.ts

```diff
@@ -0,0 +1,69 @@
+import { defineConfig, devices } from "@playwright/test";
+
+/**
+ * See https://playwright.dev/docs/test-configuration.
+ */
+export default defineConfig({
+	testDir: "./e2e-tests",
+	fullyParallel: true,
+	forbidOnly: !!process.env.CI,
+	retries: process.env.CI ? 1 : 0,
+	reporter: process.env.CI ? [["html"], ["github"]] : "html",
+
+	use: {
+		baseURL: `http://${process.env.CI ? "localhost" : "host.docker.internal"}:5173`,
```

Comment:
We need a more specific check because env vars are always string values. 
```suggestion
		baseURL: `http://${process.env.CI === "true" ? "localhost" : "host.docker.internal"}:5173`,
```

Also as per this check we are running locally on CI and docker in local dev

Reply:
IMO a truthiness check on `process.env.CI` would be sufficient, but I changed it to `process.env.CI === "true"`.

Regarding check: this is the `baseURL` the Playwright setup is using to visit the vite dev server - not where we are running things.  
But anyways, I was able to get rid of this anyways - it is now using `http://localhost:5173` in CI and locally.